### PR TITLE
feat: add internationalization capabilities

### PR DESCRIPTION
**Pull request by BryanttV**
### Description
This PR adds internationalization capabilities, and includes translations to `en-US`. Also, add the handler's messages in another module.

### CODE REVIEW

**Review by felipemontoya — APPROVED** [review]

File: sample-skill/lambda/lambda_function.py [hunk 1]
```diff
@@ -25,17 +25,36 @@
 logger.setLevel(logging.INFO)
 
 
+
+ENDPOINT = "https://lms.valencia.athena.edunext.link"
```
Comment (mariajgrimaldi) [inline sample-skill/lambda/lambda_function.py]: can this not be hardcoded here?

Comment (mariajgrimaldi) [inline sample-skill/lambda/lambda_function.py]: what's endpoint? can we be more specific with the vars naming here and in the rest of the script

**Comment by felipemontoya** [comment]
@BryanttV I found this sample code that does it similarly, but better separated in modules.

https://github.com/alexa-samples/skill-sample-python-howto/blob/master/lambda/py/alexa/data.py

You could check it out for inspiration

File: sample-skill/lambda/lambda_function.py (hunk 1 above)
Comment (BryanttV) [reply sample-skill/lambda/lambda_function.py]: I have already updated the variable names

**Comment by BryanttV** [comment]
@felipemontoya @mariajgrimaldi Guided by [this example](https://github.com/eduNEXT/openedx-alexa-adaptor-template/pull/5#issuecomment-1728490765) I added internationalization capabilities to the Skill. At the moment it can be used in English and Spanish.

File: sample-skill/lambda/alexa/data.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+"""Data for the skill."""
```
Comment (mariajgrimaldi) [inline sample-skill/lambda/alexa/data.py]: can we explain a bit more what data is?

**Review by mariajgrimaldi — APPROVED** [review]
LGTM!

**Comment by mariajgrimaldi** [comment]
Can you attach a video of the internationalization part working?

File: sample-skill/lambda/lambda_function.py (hunk 1 above)
Comment (BryanttV) [reply sample-skill/lambda/lambda_function.py]: Done

**Comment by BryanttV** [comment]
@felipemontoya, The PR has been updated. All messages were left in English by default, and the utterances were updated to include only one slot (`coursename`).

**Review by felipemontoya — APPROVED** [review]

**Pull request merged**